addon/components/article-structure-plugin/structure-card.hbs

```diff
@@ -34,7 +34,7 @@
                   @iconAlignment="left"
                   @skin="link"
                   @alert={{true}}
-                  @disabled={{this.currentStructureType.noUnwrap}}
+                  @disabled={{not this.canRemoveStructure}}
```

Comment:
Why not both? Like `(or this.currentStructureType.noUnwrap (not this.canRemoveStructure))` maybe I'm wrong, but I think that this button should be disabled if the current structure don't allow unwrapping or if it cannot be unwrapped